source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch} from '../github-helpers';
+
+function handleMenuOpening({delegateTarget: dropdown}: delegate.Event): void {
+	dropdown.classList.add('rgh-git-checkout'); // Mark this as processed
+	const [, user, repository] = select<HTMLAnchorElement>('.commit-ref.head-ref a')!.pathname.split('/', 3);
+
+	const tabContainer = select('tab-container', dropdown)!;
+	tabContainer.style.width = '370px';
+	select('.UnderlineNav-body', tabContainer)!.append(
+		<button
+			name="type"
+			type="button"
+			role="tab"
+			aria-selected="false"
+			className="UnderlineNav-item flex-1 btn-link"
+			tabIndex={1}
+		>
+			Git Checkout
+		</button>
+	);
+
+	tabContainer.append(
+		<div hidden role="tabpanel" className="p-3" tabIndex={1}>
+			<p className="text-gray text-small">
+				Run in your project repository
+			</p>
+			<div className="copyable-terminal">
+				<div className="copyable-terminal-button">
+					<clipboard-copy
+						className="btn btn-sm zeroclipboard-button"
+						role="button"
+						for="checkout-remote-help-step-1"
+						aria-label="Copy to clipboard"
+						data-copy-feedback="Copied!"
+					>
+						<ClippyIcon/>
+					</clipboard-copy>
+				</div>
+				<pre
+					id="checkout-remote-help-step-1"
+					className="copyable-terminal-content"
+				>
+					<span className="user-select-contain">git remote add {user} https://github.com/{user}/{repository}.git</span>
```

Comment:
1. This line should only appear for cross-repo PRs.
2. They should be merged into a single copyable `pre`. Even if the first one already exists and fails, the second line is still executed.
3. Pretty much never ever hard-code github.com anywhere. Use `location.origin` if necessary at all (in this case it is)